packages/account/src/providers/transaction-request/transaction-request.ts

```diff
@@ -627,7 +630,7 @@ export abstract class BaseTransactionRequest implements BaseTransactionRequestLi
       }
     };
 
-    updateAssetInput(BaseAssetId, bn(100_000_000_000));
+    updateAssetInput(this.baseAssetId, bn(100_000_000_000));
```

Comment:
This method should receive the `baseAssetId` as a parameter

Reply:
https://github.com/FuelLabs/fuels-ts/pull/1976/commits/1239a4bbaac8a320ced1c66eeab51224269e0d8d

Reply:
I agreed here, this is the only reference to `baseAssetId` as this is the only place where we'd want it specifically. https://github.com/FuelLabs/fuels-ts/pull/1976/commits/1239a4bbaac8a320ced1c66eeab51224269e0d8d